### ISSUE TITLE
StackLayout inside Scrollview with horizontal orientation not expanding

### ISSUE THREAD

**Issue by letscodewithkalyan** [dotnet/maui#25462]
### Description

When we add stack layout with horizontal orientation to Scrollview with Horizontal orientation 
StackLayout horizontal options Fill is not working

### Steps to Reproduce

1. Added below code in project 

<ScrollView Orientation="Horizontal" HeightRequest="40" BackgroundColor="Yellow">
            <StackLayout Orientation="Horizontal" BackgroundColor="Green" HorizontalOptions="Fill" VerticalOptions="Fill"/>
</ScrollView>

2. Expect Green Color in screen But Showing Yellow colour

### Link to public reproduction project repository

https://github.com/letscodewithkalyan/MauiStackLayoutIssue

### Version with bug

9.0.0-rc.2.24503.2

### Is this a regression from previous behavior?

Yes, this used to work in Xamarin.Forms

### Affected platforms

Android

### Affected platform versions

Android

**Comment by ninachen03** [comment]
I can't see the code you provided. Could you provide us with a sample project so we can investigate it further? Looking forward to your reply!

**Comment by PureWeen** [comment]
Duplicate of #24078

**Comment by michiel-nwa** [comment]
@PureWeen  I don't think this is the same issue.   This happens without any `CollectionView` being involved.

Same issue also happens with FlexLayout in a horizontal ScrollView.

e.g.
```
<ScrollView BackgroundColor="Blue" HorizontalOptions="Fill" Orientation="Horizontal">
    <StackLayout Orientation="Horizontal" BackgroundColor="Green"  HorizontalOptions="Fill">
        <Label Text="A" BackgroundColor="Aqua"  />
        <Label Text="B" BackgroundColor="Red" />
    </StackLayout>
</ScrollView>
```
and
```
 <ScrollView BackgroundColor="Blue" HorizontalOptions="Fill" Orientation="Horizontal">
    <FlexLayout Direction="Row" BackgroundColor="Green"  HorizontalOptions="Fill">
        <Label FlexLayout.Basis="100" Text="A" BackgroundColor="Aqua" FlexLayout.Grow="1"  />
        <Label FlexLayout.Basis="100" Text="B" BackgroundColor="Red" FlexLayout.Grow="1" />
    </FlexLayout>
</ScrollView>
```

This works fine on iOS (both stretch to full width there) but not on Android.

**Comment by michiel-nwa** [comment]
Workaround for now:   Set   `scrollView.Content.MinimumWidthRequest` to  `scrollView.Width`  whenever the size or `Content` changes.

**Comment by letscodewithkalyan** [comment]
@PureWeen  @ninachen03  Here is GitHub link for reproduce issue in Maui
https://github.com/letscodewithkalyan/MauiStackLayoutIssue

**Comment by letscodewithkalyan** [comment]
@samhouts @PureWeen  Can you check repo and reopen the issue and provide any workaround.

**Comment by Zhanglirong-Winnie** [comment]
This issue has been verified using  Visual Studio 17.12.0 Preview 5 (9.0.0-rc.2.24503.2 & 8.0.92 & 8.0.3). Can repro this issue on android platform. 
iOS:
i![Image](https://github.com/user-attachments/assets/fc1c8453-cea6-485a-9bc6-6aa856388b29)
Android:
![Image](https://github.com/user-attachments/assets/c4d5bc0f-6bc2-4889-9e74-4d3e51f96998)
